peerconnection.go

```diff
@@ -621,8 +621,6 @@ func (pc *PeerConnection) addAnswerMediaTransceivers(d *sdp.SessionDescription)
 			switch {
 			case t.kind != kind:
 				continue
-			case direction == RTPTransceiverDirectionSendrecv && t.Direction != RTPTransceiverDirectionSendrecv:
```

Comment:
Will this currently allow a  `RTPTransceiverDirectionSend` to satisfy a `RTPTransceiverDirectionSendrecv` ?

Reply:
In the future yes which I believe is alright, but currently no since `AddTransceiver` only supports `sendrecv` and 'recvonly` (https://github.com/pion/webrtc/blob/master/peerconnection.go#L1414).

Reply:
Will this break the other side? I need to confirm how Chromium acts.

If the other side is offering to send us video (via a SendRecv) candidate and we return with a (SendOnly) will it throw an error because we aren't able to satisfy? Or will it it gracefully just not send.

Also if we make this change we should probably prioritize giving them a `SendRecv`, `Recv` and then if we can't do anything a `Send` candidate. 

Sorry don't want to throw more work on you, our SDP parsing has to do a lot more with UnifiedPlan + multi-track send+recv so sometimes I burn a few cycles on it :)

Reply:
That's a good question. For the moment, I can update this pull request so only `sendrecv` and `recvonly` is accepted here. Once we confirm `sendonly` works, we can create a new pull request that adds this functionality.

Reply:
Updated the pull request to only accept `sendrecv` and `recvonly` here. Let me know if this looks better.